pyproject.toml

```diff
@@ -44,9 +32,36 @@ classifiers = [
     "Operating System :: OS Independent",
     "Topic :: Text Processing :: Linguistic",
 ]
+include = ["udtube*"]
+exclude = ["configs*",  "examples*", "scripts*" ]
+
+[tool.poetry.dependencies]
+python = "^3.9"
```

Comment:
Right now we have Python 3.8 support listed elsewhere. It was end-of-lifed on October 7th though so we could drop it from `pyproject.toml`.

Reply:
Okay, so I am understanding 3.8 is okay?

Reply:
yeah but change it (in two places) in `pyproject.toml` to eliminate references to 3.8 before submitting

Reply:
Sorry, I don't see any references to 3.8 in the `pyproject.toml`, or elsewhere in the project. 3.9 support and up is present in this `pyproject.toml`. I see some in the poetry.lock, but that's autogenerated. Am I misunderstanding something?

Reply:
You're right, I was seeing an old cache of things, disregard.